scmrepo/git/backend/dulwich/__init__.py

```diff
@@ -344,8 +345,11 @@ def is_tracked(self, path: str) -> bool:
         return False
 
     def is_dirty(self, untracked_files: bool = False) -> bool:
-        staged, unstaged, untracked = self.status()
-        return bool(staged or unstaged or (untracked_files and untracked))
+        kwargs: Dict[str, Any] = (
+            {} if untracked_files else {"untracked_files": "no"}
+        )
+        staged, unstaged, untracked = self.status(**kwargs)
+        return bool(staged or unstaged or untracked)
```

Comment:
`any` makes the intention more clearer.
```suggestion
        staged, unstaged, untracked = self.status(**kwargs)
        return any((staged, unstaged, untracked))
```

You can further simplify this to just:

```suggestion
        return any(self.status(**kwargs))
```

or, 
```